static/app/views/performance/transactionSummary/transactionOverview/index.tsx

```diff
@@ -79,12 +81,22 @@ function TransactionOverview(props: Props) {
         tab={Tab.TransactionSummary}
         getDocumentTitle={getDocumentTitle}
         generateEventView={generateEventView}
-        childComponent={OverviewContentWrapper}
+        childComponent={CardinalityLoadingWrapper}
       />
     </MEPSettingProvider>
   );
 }
 
+function CardinalityLoadingWrapper(props: ChildProps) {
+  const mepCardinalityContext = useMetricsCardinalityContext();
+
+  if (mepCardinalityContext.isLoading) {
+    return <LoadingContainer isLoading />;
+  }
```

Comment:
Putting a loading here which should resolve fairly fast as a stopgap for this problem. We can switch this around to be an `enabled` flag on the discover queries when we switch to RQ based discover queries here. For now it'll stop the page breaking on empty transactions for the moment (at least ones due to cardinality issues).